postcss_mailer_config/postcss.config.js

```diff
@@ -12,7 +13,7 @@ module.exports = {
     // Maybe someday we can remove this next plugin for the mailer stylesheet.
     require('postcss-css-variables'),
     require('tailwindcss/nesting'),
-    require('tailwindcss'),
+    require('tailwindcss')({ config: tailwindImportConfigFile }),
```

Comment:
Since we've "inverted" tailwindcss and postcss I had to move the reference to the tailwind config out of the CLI invocation and into the postcss config.

I also had to rename this file from `postcss.mailer.config.js` to `postcss_mailer_config/postcss.config.js` because the `--config` option to `postcss` only takes a directory name and then looks for `postcss.config.js` in that directory. There doesn't seem to be a way to specify a config file that's named anything other than `postcss.config.js`.